ingest/src/main/java/com/microsoft/azure/kusto/ingest/IngestClientImpl.java

```diff
@@ -45,9 +47,33 @@
     @Override
     public IngestionResult ingestFromBlob(BlobSourceInfo blobSourceInfo, IngestionProperties ingestionProperties)
             throws IngestionClientException, IngestionServiceException {
+        return ingestFromBlobImpl(blobSourceInfo, ingestionProperties);
+    }
+
+    @Override
+    public CompletableFuture<IngestionResult> ingestFromBlobAsync(BlobSourceInfo blobSourceInfo, IngestionProperties ingestionProperties) {
+        return CompletableFuture.supplyAsync(
+                () -> {
+                    try {
+                        return ingestFromBlobImpl(blobSourceInfo, ingestionProperties);
+                    } catch (IngestionClientException | IngestionServiceException e) {
+                        log.error("Error when ingestFromBlobAsync()", e);
+                        // Here we throw a CompletionException which extends the RuntimeException.
```

Comment:
please rephrase. logging the exception will log the stacktrace and it will include the method name.
Try ("Failed to ingest from blob: ", e)

Reply:
Changes the exception messages in all interface methods